src/rules/chromemanifest/categories.js

```diff
@@ -14,7 +14,7 @@ export const DANGEROUS_CATEGORIES = [
 ];
 
 
-export function checkCategories(triples) {
+export function checkCategories(triples, filename) {
```

Comment:
Changing this, even though filename is passed in the `triples` object, so we have a consistent rule function signature for every rule type other than ESLint/JS.